iocage_lib/ioc_start.py

```diff
@@ -1312,19 +1311,49 @@ def __check_rtsold__(self):
             stdout=su.PIPE
         )
 
-    def get_default_gateway(self):
-        # e.g response - ('192.168.122.1', 'lagg0')
-        try:
-            return netifaces.gateways()["default"][netifaces.AF_INET]
-        except KeyError:
+    def get_default_interface(self):
+        if self.host_gateways['ipv4']['interface'] != 'none':
```

Comment:
@joachimmathes you forgot to update this usage ;) It will never be `none` (str)

Reply:
I'm back from vacation. Will take care of that tomorrow.

Reply:
Fixed.